src/index.ts

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+/*
+	File: src/index.ts
+	cpuabuse.com
+*/
+
+/**
+ * Entry point for the project.
+ */
+
+/**
+ * @license ISC
+ * ISC License (ISC)
+ *
+ * Copyright 2020 cpuabuse.com
+ *
+ * Permission to use, copy, modify, and/or distribute this software for any purpose with or without fee is hereby granted, provided that the above copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+import { Command } from "commander";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Command' is defined but never used.](https://app.codacy.com/gh/cpuabuse/cross-echo/pullRequest?prid=5213902)